packages/client/src/workflow-client.ts

```diff
@@ -190,7 +199,32 @@ export interface WorkflowResultOptions {
    *
    * @default true
    */
-  followRuns: boolean;
+  followRuns?: boolean;
+}
+
+export interface GetWorkflowHandleOptions extends WorkflowResultOptions {
+  /**
+   * ID of the first execution in the Workflow execution chain.
```

Comment:
Define or link to existing definition of chain

Reply:
Do we have one?

Reply:
No